gst/nnstreamer/include/nnstreamer_plugin_api_converter.h

```diff
@@ -38,38 +38,35 @@ extern "C" {
 /**
  * @brief Converter's subplugin implementation.
  */
-struct _NNStreamerExternalConverter {
+typedef struct _NNStreamerExternalConverter {
```

Comment:
We already have ```typedef struct _NNStreamerExternalConverter NNStreamerExternalConverter;``` in tensor_converter.h. Doesn't it matter?

Reply:
Thank you. Don't need to re-define. I will revert it.